### ISSUE TITLE
EE103 text - Fehlermeldung

### ISSUE THREAD

**Issue by stefaniasalvadori** [dariok/karlstadt-issues#221]
Ich habe die Word-Datei hochgeladen, die ich am 5. November mit  Dir transformiert hatte, und bekomme nun diese Fehlermeldung:

![cattura](https://user-images.githubusercontent.com/28624747/48477879-0585c980-e803-11e8-8515-d191a0b3e4d5.PNG)

Formatierungsfehler in den textkrit. Anmerkungen?

**Comment by dariok** [comment]
Es gibt eine krit. Anm. mit falschem Kommentar:
„sactissimum a1; **korrigiert in** sanctissumum a2“

Bei einer fehlt eine Kursivierung des Originaltextes (vor „Querit clariss'imus' d'ominus' d'octor' quam habeat...“)
„Eckius. A4-5; **Eckius** a1; de activitate ferule a2 AuRd-Gl“

**Comment by stefaniasalvadori** [comment]
Beide Stellen korrigiert.
Nun bekomme ich diese Fehlermeldung:
![cattura](https://user-images.githubusercontent.com/28624747/48489860-493cfb00-e824-11e8-82b3-fc607f56dff5.PNG)

**Comment by dariok** [comment]
Ah, da hat Word aus „1-4“ (mit Divis) „1–4“ (mit Halbgeviert) gemacht; für den Druck wäre das richtig, nur für das XML muß ich es anpassen.
Bitte einmal ausprobieren.

**Comment by stefaniasalvadori** [comment]
quasi!
Nun ist die Angabe (?) in den textkrit. Anm. problematisch
Wenn ich die runden Klammer lösche wäre es besser?

![cattura](https://user-images.githubusercontent.com/28624747/48497146-dcc9f800-e833-11e8-99c1-241d551acbcc.PNG)

**Comment by dariok** [comment]
Auch das Fragezeichen alleine wird problematisch sein – was soll es denn hier bedeuten?
Ggf. wäre es als Kommentar besser.

**Comment by dariok** [comment]
Ich muß mal schauen, wie ich das umsetze. Du brauchst es aber nicht löschen.
Wenn es sich nicht direkt umsetzen läßt, dann sage ich Dir nochmal bescheid, einen Moment.

**Comment by dariok** [comment]
Kommentar sollte immer funktionieren.

_sum_ über der Zeile hinzugefügt a₃ (?)

heißt also, daß die Lesung "sum" nicht sicher ist?

**Comment by stefaniasalvadori** [comment]
nein, es heiss das unsicher ist, ob a3 oder a1 _sum_ geschrieben hat - in diesen Stellen kann (?) weg

Wäre: _sum_ (?) über der Zeile hinzugefügt a₃ würde heissen: _sum_ ist nicht sicher - diese Stellen könnte ich mit ein Kommentar umformulieren "unlesbar" "kaum lesbar" oder similia

**Comment by dariok** [comment]
Ah, okay.
Es sollte reichen, (?) als Kommentar zu markieren.
Ich habe noch einen Punkt gefunden, den ich gleich angehe; ich habe es aber vorläufig einmal manuell hochgeladen, damit Du es Dir anschauen kannst.

**Comment by stefaniasalvadori** [comment]
Ok, das heisst, ich markiere nun alle (?) als Kommentar und danach lade ich die EE hoch?
Oder soll ich abwarten?

**Comment by stefaniasalvadori** [comment]
no stress! 
Whenever you want

**Comment by dariok** [comment]
Es gibt eine FN

ad hoc ut <...> aii _a₁_

Hier ist das Spatium vor "ut" kursiv, müßte aber recte stehen

**Comment by dariok** [comment]
Ende C3v, hinter "Quintodecimo": bei "Decimo 5to" müßte das hochgestellte "to" auch kursiviert werden.

Und dann versuchen, hochzuladen.

**Comment by stefaniasalvadori** [comment]
fast... auch 1-5 ist problematisch?
Soll ich alle ähnliche Stellen Dir nennen?

![cattura](https://user-images.githubusercontent.com/28624747/48515751-e0747380-e861-11e8-9f5f-a36dbcb561c0.PNG)

**Comment by dariok** [comment]
Ja, das ist ein Fall, wo der witness teils recte ist (B3r, "sunt deo A")

Ich habe an dieser Stelle nun eingebaut, daß vor der Zahl "ZZ" ausgegeben wird. Wenn Du also "ZZ" in einer ANmerkung findest, dann ist der Buchstabe des Witness nicht kursiv.

Du kannst es dann nochmal versuchen.

**Comment by stefaniasalvadori** [comment]
hier nochmals:
![cattura](https://user-images.githubusercontent.com/28624747/48517211-e66c5380-e865-11e8-9737-92328110fcff.PNG)

**Comment by stefaniasalvadori** [comment]
Ich habe die Stelle gefunden - A5Lp nicht kursiviert - ich lade nochmals hoch

**Comment by dariok** [comment]
B3r, Ende

Ja, für diesen Fall werde ich jetzt auch ZZ ausgeben; damit sollte es sich besser finden lassen

**Comment by stefaniasalvadori** [comment]
hier nun dies - das kann ich nicht nachvollziehen leider..

![cattura](https://user-images.githubusercontent.com/28624747/48517511-d1dc8b00-e866-11e8-844b-22c09ee89a57.PNG)

**Comment by dariok** [comment]
C3v, "in se est": In den FN steht "feht" statt "fehlt"

**Comment by stefaniasalvadori** [comment]
Siiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii!!!!!
Es funktioniert!
GRANDIOSO

Ich habe auch gleich die nicht kursivierten Witnesses kursiviert - nun sollten alle Witnesses richtig transformiert und kodiert

Ich weine... :-)

Gut, wir können auch die PDF dieser EE herstellen (die intro lade ich in kurze hoch)
Danke nochmals Dario!

**Comment by stefaniasalvadori** [comment]
Die Einleitung der Leipziger Disputation habe ich hochgeladen - sieht sehr gut aus!
Es gibt nur eine Wiederholung der Anmerkungen der Beschreibung des Manuskripts (im Ueberlieferung). Im Text springt von Anm.31 zu Anm. 99 - im Apparat  werden alle Anm. verdoppelt.
Dass können wir aber in den kommenden Tagen klären oder manuell in xml korrigieren (scheint der Text auch verdoppet zu sein):

![cattura](https://user-images.githubusercontent.com/28624747/48519017-b1fb9600-e86b-11e8-8f45-acf40341400c.PNG)

[EE103intro2018_11-14.docx](https://github.com/dariok/karlstadt-issues/files/2583029/EE103intro2018_11-14.docx)


Für heute einfach und nochmals danke!

**Comment by dariok** [comment]
Okay, ich glaube ich habe den Grund gefunden. Muß jetzt noch testen.

**Comment by dariok** [comment]
Ja, paßt jetzt.

**Comment by stefaniasalvadori** [comment]
Danke Dario!
Online ist die ganze Ueberlieferung nicht mehr angezeigt (nur Handschrift-signatur und literatur).
Soll/Darf ich die Einleitung nochmals hochladen, oder nicht?

**Comment by dariok** [comment]
Ja, kannst Du gerne nochmal hochladen. Ich denke, daß es online nur ein Anzeigefehler ist.

**Comment by stefaniasalvadori** [comment]
Ok, es klappt
Nur die QV zwischen Anm. funktionieren nicht - in Word funktionieren sie

**Comment by stefaniasalvadori** [comment]
ok, keine Eile

**Comment by dariok** [comment]
Das Problem mit den Verweisen sollte auch erledigt sein; bitte die Einleitung einmal neu hochladen.
